config/field_test.yml

```diff
@@ -5,11 +5,13 @@ experiments:
       - more_random
       - more_tag_weight
       - mix_base_more_random
+      - more_tag_weight_more_random
     weights:
-      - 65
+      - 55
       - 10
       - 10
       - 15
+      - 10
```

Comment:
We might want to up this a bit next week: I don't have a sense of how much the percentages matter here